mlrun/runtimes/daskjob.py

```diff
@@ -292,6 +292,7 @@ def close(self, running=True):
 
         try:
             client = default_client()
+            client.shutdown()
```

Comment:
It's not clear that `client.shutdown()` is affecting the cluster and not the client, so I suggest adding a comment above 
`# Shutdown the cluster first, then close the client`
(to save who ever reading this code from having to go inside both functions (close and shutdown) to understand the difference between them)